### ISSUE TITLE
Can't work with pixi v5.2

### ISSUE THREAD

**Issue by Vi-jay** [pixijs/pixi-projection#60]
// not work
```js
const PIXI = require("pixi.js");
window.PIXI = PIXI;
require("pixi-projection");
```
// I changed this. it's worked
```js
const PIXI = require("pixi.js/dist/pixi"); //I changed the export module for this file.
window.PIXI = PIXI
require("pixi-projection");
```

**Comment by ivanpopelyshev** [comment]
Interesting. Thanks, i'll check it.

**Comment by Vi-jay** [comment]
If you fix this problem, please let me know, thank you very much.

**Comment by K024** [comment]
I met the save problem.
Here it is. 

[BatchRenderer refactoring to increase performance (#6162)](https://github.com/pixijs/pixi.js/commit/6d66be14e67e5f3219831c558893cbb5fe8ff790)
> PIXI.BaseTexture._id batch local field has been changed to `_batchLocation`

You can use `PIXI.useDeprecated()` as a temporary solution, or fix it in [`Sprite2dRenderer`](https://github.com/pixijs/pixi-projection/blob/2057787b70d7bdfdcb8fd1514520d3faddaf25ea/src/base/webgl/Sprite2dRenderer.ts#L95).

**Comment by ivanpopelyshev** [comment]
OK, right, according to examples - it actually works with 5.2, it only produces one deprecation. Yes, I know about it , I was the one who put that deprecation there. 

@Vi-jay 
What does not work for you?

**Comment by Shyiy** [comment]
For me it also doesn't work if paired with Pixi 5.2.
More specifically if I try to project the following simple sprite:
```
const backgroundSprite: PIXI.projection.Sprite2d = new PIXI.projection.Sprite2d(PIXI.Texture.WHITE);
backgroundSprite.tint = 0xff0000;
backgroundSprite.alpha = 0.25;
```
via the `mapSprite` method, nothing is displayed. The same code works correctly with older versions of Pixi (5.1.5).

**Comment by ivanpopelyshev** [comment]
But but but.. it works here: https://pixijs.io/examples/?v=dev#/plugin-projection/basic.js  , with dev and with 5.2.0 , just fine.

I believe pixi-projection in examples is the same as here.

How can it not work in your case?

I'm gonna publish new version in weekend but I really don't know whether it will work for you :( I really need more information.

**Comment by ivanpopelyshev** [comment]
Do you see that deprecation in console, @Shyiy ? maybe you use pixi without deprecations somehow..

**Comment by Shyiy** [comment]
It seems to be somehow related to imports/require.
I use [parcel.js](https://parceljs.org/), and if I try to set up a clean project containing only the same code as in the demo at https://pixijs.io/examples/?v=v5.2.0#/plugin-projection/quad-homo.js, with on top the following to import and use PIXI and projection, then it doesn't show the projection as explained, only the 4 vertices to be dragged around. No message in the console.

```
import * as PIXI from "pixi.js";
globalThis.PIXI = PIXI;
import "pixi-projection";
```

![Screenshot 2019-11-29 at 17 36 22](https://user-images.githubusercontent.com/6885349/69881913-c96abd00-12ce-11ea-99e7-167200126dc7.png)

**Comment by ivanpopelyshev** [comment]
try `require('pixi-projection')` instead.

**Comment by bigtimebuddy** [comment]
If you are bundling, try calling this method to enable deprecated settings:

http://pixijs.download/dev/docs/PIXI.html#.useDeprecated

**Comment by Shyiy** [comment]
Unfortunately none of the solutions seem to change things (still not working and no error in the console):
```
const PIXI = require("pixi.js"); // also tried like before with import * as PIXI from "pixi.js";
globalThis.PIXI = PIXI;
PIXI.useDeprecated();
require("pixi-projection");
```

**Comment by ivanpopelyshev** [comment]
OK, special for you, I made half-cooked 3.6 that does have `_id` swapped to `_batchLocation`, thus fully compatible with 5.2 without deprecations.

If it doesn't work - try remove node_modules and package.lock and reinstall everything from scratch.
If it doesn't work again - then sorry, I we dont support your webpack configuration, that is `unknown configuration`, that you didnt specify.

**Comment by Shyiy** [comment]
Thanks for the quick support. That's actually not needed since as I reported before, enabling the deprecated settings makes it work correctly now.
The problem was caused by my bundler combined with myself not knowing about the http://pixijs.download/dev/docs/PIXI.html#.useDeprecated function that needed to be called to include the needed deprecations. Without calling it the pixi-projection library would just silently fail.
